### ISSUE TITLE
A more thorough  guide 

### ISSUE THREAD

**Issue by PK98** [kartikey54/VaccineAvailabilityNotifier#11]
Hi, first of all, hats off and thank you so much for the effort. 
I just have one request if you could elaborate in step by step details on how to use the script (starting from something as simple as how to download it all and then running it) as I don't have much idea on how to use git and then run the desired set of codes. I feel many others with weaker or no technical background might also face the same issue. Additional details would be really helpful. Thank you so much once again.

**Comment by rahulchungath** [comment]
Yes exactly. Please shed more details on how to run this. I got till the Google verification part, but i'm not sure how to give access too the application.  But thank you for the initiative.

**Comment by ghaisasadvait** [comment]
What does the 'Application Password' field in the .env file mean?.. Since cowin uses otp for authentication, i was confused.
